samples/matter/common/src/bridge/matter_bridged_device.h

```diff
@@ -46,20 +76,27 @@ class MatterBridgedDevice {
 		TemperatureSensor = 0x0302,
 		HumiditySensor = 0x0307
 	};
-
+	using IdentifyType = chip::app::Clusters::Identify::IdentifyTypeEnum;
 	static constexpr uint8_t kDefaultDynamicEndpointVersion = 1;
 	static constexpr uint8_t kNodeLabelSize = 32;
 	static constexpr uint8_t kDescriptorAttributeArraySize = 254;
 
 	explicit MatterBridgedDevice(const char *nodeLabel)
+		: mIdentifyServer(mEndpointId, IdentifyStartDefaultCb, IdentifyStopDefaultCb,
+				  IdentifyType::kVisibleIndicator)
```

Comment:
Could we create an mIdentifyType field in this class and assign `kVisibleInditcatior` to it? I know for our examples we are not going to use other type of indicator, but if customer would like to use something else, it will not be possible without core code modification with the current implementation.

Reply:
I guess, we can just set it to kNone and let the developer to select the supported mode in the concrete device implementation. I chose kVisibleInditcatior, because I saw it was the solution suggested by the Matter spec somewhere. But then I'm not sure if the identify-server will process the identify command as expected, I will check that.

Anyway, `ConfigureIdentifyServer()` method can be implemented in the concrete device class to overwrite the default identify-server configuration.

Reply:
Yeah, but it works the same way for all other attributes. The user could also bypass our bridged device layer and just set the cluster value directly. I just think it may not be obvious for the user why we skip representing one selected attribute in the class, while we represent the other ones.

Reply:
> I chose kVisibleInditcatior, because I saw it was the solution suggested by the Matter spec somewhere.

I'm not against it, I just think it should not be hardcoded in the constructor, but selected as a default value for the class field.

Reply:
> Yeah, but it works the same way for all other attributes. The user could also bypass our bridged device layer and just set the cluster value directly. I just think it may not be obvious for the user why we skip representing one selected attribute in the class, while we represent the other ones.

But which attributes are you referring to? The IdentifyType is represented already in the Identify instance object which is not the case for IdentifyTime, therefore we need to keep in the class to handle it as an external attribute (support reading and writing). I assume the user needs to use ConfigureIdentifyServer(Identify*) method which provides the direct access to the Identify instance to align Identify configuration. I get your point, but in the current implementation the IdentifyTime kept as a field would never change its value anyway.

Reply:
Ok, in my opinion we create sort of interface for managing the dynamic endpoints and their clusters. If something can be customized by the user, we provide a possibility to set it, without invoking directly the clusters code.

For example, the device node label is typically something that is assigned during object creation and will not be modified ever again. Despite that fact we do not hardcode it, but we let user set some custom value (in this particular case using constructor).

I would imagine we should do the similar thing in this scenario. The IdentifyType is something constant at device life time, but not something what cannot be changed by the customer (e.g. ClusterRevision). Of course, if the mIdentifyType object exists somewhere we do not have to duplicate it, but I would expect we should allow to set its value in the constructor (maybe not the best option imo) or provide a Set method that would call  `ConfigureIdentifyServer(Identify*)` underneath. I just feel that providing API to customize all attributes but one that has to be modified using cluster specific code is not quite right.

Reply:
Discussed F2F